### PR TITLE
fix(spaces): update Spaces page layout

### PR DESCRIPTION
**Pull request by mindreeper2420**
update the Spaces page to have scrolling description, even sizes cards and proper button links

### CODE REVIEW

**Comment by pmuir** [comment]
[test]

File: src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html [hunk 1]
```diff
@@ -3,6 +3,11 @@
     <h2 class="card-pf-title">{{space.attributes.name}}</h2>
   </div>
   <div class="card-pf-body">
-    <div almEditable [editable]="isEditable() | async" (onUpdate)="saveDescription($event)">{{space.attributes.description}}</div>
+    <p>
+      <b>Description</b>
```
Comment (pmuir) [inline src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html]: Edit symbol and title show on a description you *can't* edit, which I don't think is correct.

**Comment by joshuawilson** [comment]
@mindreeper2420 please address @pmuir comment and rebase

**Comment by mindreeper2420** [comment]
@pmuir updated to account of editable vs non-editable description

@joshuawilson rebased

**Pull request merged**